src/services_models/GroupReserve.jl

```diff
@@ -0,0 +1,92 @@
+struct GroupReserve <: AbstractReservesFormulation end
+############################### Reserve Variables` #########################################
+"""
+This function checks if the variables for reserves were created
+"""
+function check_activeservice_variables(
+    psi_container::PSIContainer,
+    contributing_services::Vector{<:PSY.Service},
+)
+    for service in contributing_services
+        name = PSY.get_name(service)
+        var =
+            get(psi_container.variables, make_variable_name(name, typeof(service)), nothing)
```

Comment:
I changed the error to mention that it is a "contributing service" to help the user see where it came from. Do you guys think that this is a good practice? Do you have a suggestion on how to use an accessor function here?

Reply:
That could be helpful, but the stacktrace for the exception will include this function/line, so we can figure it out.  I understand that that is not as obvious as a simple error message like this.  If you prefer having your own log message then you could catch the exception, log the error, and then rethrow.

Here is the accessor function.
```
get_variable(psi_container, name, typeof(service))
```

@jd-lara We need something documented in the code about how to create and retrieve variables.  I get confused about how some names are based on the component type and some include component names.  It will take a while to make the entire code base consistent, but at least new code can follow the guidance.

Reply:
@daniel-thom correct we have some legacy code about the creation and retrieval of variables that needs consistency. @andrewrosemberg, the accessor Dan mentioned is the correct one to use in this context.

Reply:
fair enough, yeah the stack trace already does that.  Just changed back to `get_variable`!